src/vmod_xkey.c

```diff
@@ -220,6 +220,8 @@ static struct xkey_oc *
 xkey_oc_new(void)
 {
 	struct xkey_oc *oc;
+	vsc->g_oc_bytes += sizeof(*oc);
```

Comment:
is it the only code change needed here? All others changes seem to shuffle things around without changing the logic

Reply:
Yes. We wanted to put the increments and decrements in a consistent place for all the metrics.